sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/externalrules/CxxExternalRuleRepository.java

```diff
@@ -58,8 +58,16 @@ public void define(Context context) {
         }
       }
     }
-
-    //i18nLoader.load(repository); //@todo?
+    
+    for (NewRule rule : repository.rules()) {
+      rule
+        //FIXME: set internal key to key to ensure rule templates works properly : should be removed when SONAR-6162 is fixed.
+        .setInternalKey(rule.key())
+        // set default Technical Debt values for 'other' rules
+        .setDebtSubCharacteristic("ERRORS")
```

Comment:
i think we should not do this, setting these defaults values are going to cause more issues than not. Because most issues from other rules are not belonging any "default". Same goes for time to fix. adding default values like you are over or uderestimating everything that goes into sonar

I tought you would more including a parser for users to define their own values, imo that would be more usefull.